### ISSUE TITLE
Sorting with filters applied

### ISSUE THREAD

**Issue by saynala** [mleibman/SlickGrid#1166]

**Comment by saynala** [comment]
When using dataview, filter and custom compared, is there a way to sort just the group headers without reordering the group members

**Comment by 6pac** [comment]
I don't think so. This might be a good optimisation, though. Sorting is quite fast, you might not need it. All happens in the browser.
There are a few optimisations for the filtering, you could look at those.

**Comment by 6pac** [comment]
BTW, use https://github.com/6pac/SlickGrid, this repo is not maintained.
